aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/forms/controls/MultiSelectInput.get.desc.xml

```diff
@@ -0,0 +1,6 @@
+<webscript>
+  <shortname>MultiSelectInput Test</shortname>
```

Comment:
We need to make these names and descriptions more useful as they're now displayed in the main unit test page, e.g. "MultiSelectInputFormControl" as the shortName, and something along the lines of "Shows an example of the alfresco/forms/controls/MultiSelectInput form control using mocked tag data"